CONTRIBUTING.md

```diff
@@ -62,8 +62,14 @@ We follow the [AirBnB JavaScript style guide](https://github.com/airbnb/javascri
 
 ## Licensing
 
-Unless otherwise specified, all Embark open source projects are licensed under a dual MIT OR Apache-2.0 license, allowing licensees to chose either at their option. You can read more in each project's respective README.
+Unless otherwise specified, all Embark open source projects shall comply with the Rust standard licensing model (MIT + Apache 2.0) and are thereby licensed under a dual license.
```

Comment:
```suggestion
Unless otherwise specified, all Embark open source projects shall comply with the Rust standard licensing model (MIT + Apache 2.0) and are thereby licensed under a dual license, allowing licensees to choose either MIT OR Apache-2.0 at their option.
```